source/index.ts

```diff
@@ -0,0 +1,746 @@
+import fs = require('fs');
+import path = require('path');
+import crypto = require('crypto');
+import assert = require('assert');
+import EventEmitter = require('events');
+import dotProp = require('dot-prop');
+import makeDir = require('make-dir');
+import pkgUp = require('pkg-up');
+import envPaths = require('env-paths');
+import writeFileAtomic = require('write-file-atomic');
+import Ajv = require('ajv');
+import debounceFn = require('debounce-fn');
+import semver = require('semver');
+import onetime from 'onetime';
+
+const plainObject: () => object = () => Object.create(null);
```

Comment:
Typo

Reply:
I don't see any typo in this line Sindre. What did you mean?

Reply:
No idea. Must have been a mistake on my part.